class/defaults.yml

```diff
@@ -1,3 +1,8 @@
 parameters:
   prometheus_pushgateway:
     namespace: syn-prometheus-pushgateway
+    prometheus_rule_labels:
+      prometheus: main
```

Comment:
Maybe we should default to `prometheus: platform` which has been the prometheus name until now.